models/renegadeXStats.go

```diff
@@ -58,14 +59,99 @@ type RenegadeXStats struct {
 
 func (statsData *RenegadeXStats) ParseJsonData(rawData map[string]interface{}) error {
 	fmt.Println("Attempting to parse raw data")
+
+	//Who won this game? integer represents teams
 	winner, err := strconv.Atoi(rawData["winner"].(string))
 	if err != nil {
 		return err
 	}
 	statsData.Winner = int16(winner)
+	//What was the win method? should be a string
+	winMethod := rawData["winMethod"].(string)
+
+	if winMethod == "" {
+		return errors.New("win method should exist")
+	}
+	statsData.WinMethod = winMethod
+
+	//Parse team data for team 0
```

Comment:
You can put the code until line 94 in a function to prevent code repetition for each team